docs/sources/mimir/get-started/play-with-grafana-mimir/index.md

```diff
@@ -1,4 +1,7 @@
 ---
+aliases:
+  - /tutorials/play-with-grafana-mimir/
+  - /docs/mimir/latest/tutorials/play-with-grafana-mimir/
```

Comment:
This will need to be relative so that it doesn't steal redirects from future versions if this file is moved again.

Reply:
@jdbaldry, please have another look.